specification/spec.md

```diff
@@ -0,0 +1,113 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# BIDS Stats Models Specification
+
+Editors:
+: [Alejandro de la Vega][] ([University of Texas at Austin][])
+: [Christopher J Markiewicz][] ([Stanford University][])
+
+---
+
+## Abstract
```

Comment:
```suggestion
```

I think you can leave out this header and just put the abstract after the title, like I did on the main page